pymzml/run.py

```diff
@@ -285,10 +350,10 @@ def _init_iter(self):
             elif element.tag.endswith('}cv'):
                 if not self.info['obo_version'] \
                         and element.attrib.get('id', None) == 'MS':
-                    self.info['obo_version'] = element.attrib.get(
-                        'version',
-                        '1.1.0'
-                    )
+                    obo_in_mzml = element.attrib.get('version', '1.1.0')
+                    print('obo_in_mzml', obo_in_mzml)
```

Comment:
Please remove the print statement

Reply:
No problem 👍

Reply:
I think I removed all print lines from my side, please have a look.